src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/ChangePrice.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.Apprentice.ChangePriceViewModel
+
+@{
+    ViewBag.Title = "What's the new agreed apprenticeship price?";
+    ViewBag.GaData.Vpv = "/apprentices/apprentice/details";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
```

Comment:
Shall we use this story as a chance to change the error message in the shared partial to: `There is a problem`?